src/refract/Registry.h

```diff
@@ -10,32 +10,31 @@
 
 #include <map>
 #include <string>
+#include <memory>
+
+#include "ElementIfc.h"
 
 namespace refract
 {
-
-    // Forward declarations of IElement
-    struct IElement;
-
     class Registry
     {
         // FIXME: potentionally dangerous,
         // if element is deleted and not removed from registry
         // solution: std::shared_ptr<> || std::weak_ptr<>
-        typedef std::map<std::string, IElement*> Map;
+        typedef std::map<std::string, std::unique_ptr<IElement> > Map;
```

Comment:
We probably can remove comment above. Is it correct?

Reply:
yeah this can be removed